<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>465</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>861</width>
    <height>465</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>天气查询</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_11">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>日  期：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_dateTime">
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="cBox_province">
         <property name="minimumSize">
          <size>
           <width>69</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>省</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cBox_city">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>市</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cBox_area">
         <property name="minimumSize">
          <size>
           <width>75</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>区/县</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_refresh">
         <property name="text">
          <string>查  询</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>当日天气</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="3">
            <widget class="QTableWidget" name="tableWidget">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>56</height>
              </size>
             </property>
             <property name="columnCount">
              <number>8</number>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
             <column/>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QGroupBox" name="groupBox_1">
             <property name="title">
              <string>感冒指数</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QLabel" name="label1_1">
                  <property name="minimumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label1_2">
                  <property name="minimumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>防晒指数</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label2_1">
                  <property name="minimumSize">
                   <size>
                    <width>206</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>206</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label2_2">
                  <property name="minimumSize">
                   <size>
                    <width>206</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>206</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QGroupBox" name="groupBox_3">
             <property name="title">
              <string>穿衣指数</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLabel" name="label3_1">
                  <property name="minimumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label3_2">
                  <property name="minimumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>运动指数</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label4_1">
                  <property name="minimumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label4_2">
                  <property name="minimumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>洗车指数</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label5_1">
                  <property name="minimumSize">
                   <size>
                    <width>206</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>206</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label5_2">
                  <property name="minimumSize">
                   <size>
                    <width>206</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>206</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QGroupBox" name="groupBox_6">
             <property name="title">
              <string>晾晒指数</string>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout">
                <item>
                 <widget class="QLabel" name="label6_1">
                  <property name="minimumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label6_2">
                  <property name="minimumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>207</width>
                    <height>55</height>
                   </size>
                  </property>
                  <property name="wordWrap">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>未来天气</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QTableWidget" name="tableWidget_forecast">
           <property name="columnCount">
            <number>7</number>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>历史天气</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QTableWidget" name="tableWidget_history">
           <property name="columnCount">
            <number>8</number>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
           <column/>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
